src/fides/api/util/logger_context_utils.py

```diff
@@ -103,7 +103,12 @@ def request_details(
         LoggerContextKeys.url.value: prepared_request.url,
     }
     if CONFIG.dev_mode and prepared_request.body is not None:
-        details[LoggerContextKeys.body.value] = prepared_request.body
+        if isinstance(prepared_request.body, bytes):
+            details[LoggerContextKeys.body.value] = prepared_request.body.decode(
+                "utf-8"
+            )
```

Comment:
Now that the body can be `bytes`, we need to decode it so we can output it to the logs.